src/test/resources/project/untitled/src/main/java/ArrayUtils.java

```diff
@@ -1,5 +1,11 @@
 /**
- * (From Apache Commons Lang: https://github.com/apache/commons-lang/blob/d2687419c6973572d1621afc1b8546f5262769c3/src/main/java/org/apache/commons/lang3/ArrayUtils.java#L2325)
+ * (From Apache Commons Lang: https://github.com/apache/commons-lang/blob/d2687419c6973572d1621afc1b8546f5262769c3/src/main/java/org/apache/commons/lang3/
```

Comment:
Are the changes necessary here?

Reply:
No, I did not even know they were there until you told me. I have fixed it and now they are as they were before.